clinica/utils/input_files.py

```diff
@@ -457,20 +457,27 @@ def pet_volume_normalized_suvr_pet(
 # pet-linear
 
 
-def pet_linear_nii(acq_label, suvr_reference_region, uncropped_image):
+def pet_linear_nii(acq_label, suvr_reference_region, uncropped_image, use_uniform):
```

Comment:
To avoid breaking compatibility with old code.

```suggestion
def pet_linear_nii(acq_label, suvr_reference_region, uncropped_image, use_uniform: bool = False):
```